massa-client/src/repl.rs

```diff
@@ -1,48 +1,28 @@
 // Copyright (c) 2022 MASSA LABS <info@massa.net>
 
-use crate::cmds::{Command, ExtendedWallet};
-use crate::rpc::Client;
-use crate::settings::SETTINGS;
 use crate::utils::longest_common_prefix;
+
```

Comment:
The following import would be removed at the end of the journey!

Reply:
You mean this are the parts you wish to externalize ?